src/otx/cli/utils/jsonargparse.py

```diff
@@ -0,0 +1,152 @@
+"""Functions related to jsonargparse."""
+
+# Copyright (C) 2023 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+from __future__ import annotations
+
+import ast
+from contextlib import contextmanager
+from pathlib import Path
+from typing import Any, Iterator, TypeVar
+
+import docstring_parser
+from jsonargparse import ActionConfigFile, ArgumentParser, Namespace, dict_to_namespace
+
+
+def get_short_docstring(component: TypeVar) -> str:
+    """Get the short description from the docstring.
+
+    Args:
+        component (object): The component to get the docstring from
+
+    Returns:
+        Optional[str]: The short description
+    """
+    if component.__doc__ is None:
+        return ""
+    docstring = docstring_parser.parse(component.__doc__)
+    return docstring.short_description
+
+
+def flatten_dict(config: dict, parent_key: str = "", sep: str = ".") -> dict:
+    """Flatten a nested dictionary into a single-level dictionary.
+
+    Args:
+        d (dict): The dictionary to be flattened.
```

Comment:
```suggestion
        config (dict): The dictionary to be flattened.
```

Reply:
Thanks, but i already done - https://github.com/openvinotoolkit/training_extensions/pull/2805/commits/3679a70296e06f1ed7c5fd6fab1899bf65fcf023

Reply:
the docstring seems to be showing `d` for some reason. Can you double check if the docstring is updated as well?